spec/order_spec.rb

```diff
@@ -0,0 +1,10 @@
+require 'order'
+
+describe Order do
+  subject(:order) { described_class.new(restaurant: "fishnchips.csv") }
+  describe '#add' do
+    it 'adds dishes and quantities to the order hash' do
+      expect(order.add('Fish', 3)).to eq({ 'Fish' => 3 })
+    end
+  end
+end
```

Comment:
Could we create a test for receipt method to test the method is working correctly?